MANUAL.md

```diff
@@ -61,9 +61,9 @@ run `rclone -h`.
 linux binary downloaded files install example
 -------
 
-    wget http://downloads.rclone.org/rclone-v1.17-linux-amd64.zip
-    unzip rclone-v1.17-linux-amd64.zip
-    cd rclone-v1.17-linux-amd64
+    wget http://downloads.rclone.org/rclone-current-linux-amd64.zip
+    unzip rclone-current-linux-amd64.zip
+    cd rclone-current-linux-amd64
```

Comment:
The current archive unfortunately has the version name in the subdirectory.  So you could make this line

    cd rclone-v*-linux-amd64

And it would work for everything.

Or alternatively use the `-j` flag on unzip, but you probably want to make a temp directory first.